app_clients/pod-delete/pod-delete.yml

```diff
@@ -0,0 +1,52 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pod-delete
+spec:
+  template:
+    metadata:
+      labels:
+        app: pod-delete
+    spec:
+      #nodeName:
+        ## mandatory: provide name of host where target container resides
+        ## kubernetes.io/hostname: ""
+      restartPolicy: Never
+      serviceAccountName: pod-delete-sa
```

Comment:
can we make this variable dynamic?

Reply:
You mean to take it from job ENV and pass it in this container?

Reply:
Yes, In other experiments, we are doing the same

Reply:
Ah... We don't need `serviceAccount` here it is already mentioned in the pod yaml in chaoslib ...  Removing it